src/REL/ner/bert_wrapper.py

```diff
@@ -0,0 +1,11 @@
+from transformers import AutoTokenizer, AutoModelForTokenClassification
+from transformers import pipeline
+
+def load_bert_ner(path_or_url):
```

Comment:
This function returns None in case a bert model is loaded that crashes for whatever reason. Wouldn't this cause crashes or silent errors later on? I think it would be better to crash here if `path_or_url` does not exist.

Reply:
Adopted: crash without loaded model